test/indexOf.js

```diff
@@ -3,25 +3,13 @@
 var S = require('..');
 
 var eq = require('./internal/eq');
-var throws = require('./internal/throws');
 
 
 test('indexOf', function() {
 
   eq(typeof S.indexOf, 'function');
   eq(S.indexOf.length, 2);
-
-  throws(function() { S.indexOf('x', null); },
-         TypeError,
-         'Invalid value\n' +
-         '\n' +
-         'indexOf :: a -> List a -> Maybe Integer\n' +
-         '                ^^^^^^\n' +
-         '                  1\n' +
-         '\n' +
-         '1)  null :: Null\n' +
-         '\n' +
-         'The value at position 1 is not a member of ‘List a’.\n');
+  eq(S.indexOf.toString(), 'indexOf :: a -> List a -> Maybe Integer');
```

Comment:
Ah yes, I vaguely remember the discussions we had about `List` a few months ago now.